sources/packages/api/src/route-controllers/application/models/application.model.ts

```diff
@@ -269,3 +271,11 @@ export const transformToApplicationDetailDto = (
     programYearEndDate: applicationDetail.programYear.endDate,
   };
 };
+
+/**
+ * Enum for COE enrollment period.
+ */
+export enum EnrollmentPeriod {
+  Current = "current",
+  Upcoming = "upcoming",
+}
```

Comment:
I think enums in typescript should be something like this:

export enum EnrollmentPeriod {
Current = "CURRENT",
Upcoming = "UPCOMING",
}

Reply:
I do not prefer an upper case value here because it is going to be used as part of API url.